io/io/src/TBufferMerger.cxx

```diff
@@ -62,9 +66,15 @@ std::shared_ptr<TBufferMergerFile> TBufferMerger::GetFile()
 
 size_t TBufferMerger::GetQueueSize() const
 {
+   std::lock_guard<std::mutex> lock(fQueueMutex);
    return fQueue.size();
 }
 
+size_t TBufferMerger::GetBuffered() const
+{
+   return fBuffered;
+}
+
```

Comment:
Why is this function outlined?

Reply:
because at one point in time it had a lock rather than an atomic.